include/session/session.js

```diff
@@ -296,16 +296,14 @@ module.exports = function SessionModule(pb) {
      * @return {string} Session Id if available NULL if it cannot be found
      */
     SessionHandler.getSessionIdFromCookie = function(request){
-
-        var sessionId = null;
-        if (request.headers[SessionHandler.COOKIE_HEADER]) {
-
-            // Discovered that sometimes the cookie string has trailing spaces
-            for(var key in request.headers[SessionHandler.COOKIE_HEADER]){
-                if(key.trim() == 'session_id'){
-                    sessionId = request.headers[SessionHandler.COOKIE_HEADER][key];
-                    break;
-                }
+        let sessionId = null;
+
+        let cookieHeader = request.headers[SessionHandler.COOKIE_HEADER];
+        if (cookieHeader) {
+            let key = Object.keys(cookieHeader).find(x => x.trim() === SessionHandler.COOKIE_NAME) ||
+                Object.keys(cookieHeader).find(x => x.trim() === 'session_id');
```

Comment:
Is there a reason to iterate 2x here?

Reply:
yes!  While we are in the process of migration we must first check to see if they have the new cms_tn session id.  If not, we want to try and fall back to the old session_id.  We can remove this code after 24 hours when all sessions have expired.